stackoverlay.js

```diff
@@ -69,9 +70,34 @@ export function multimonitorDragDropSupport() {
 export function enableMultimonitorDragDropSupport() {
     pointerWatch = PointerWatcher.getPointerWatcher().addWatch(100,
```

Comment:
Is this 100ms delay is noticeable?

I found this in gnome shell where they set it to `1000 / 60` i.e. 60 times per second: https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/ui/magnifier.js#L229

Reply:
Given we're only using this for drag/drop across monitors, or detecting when mouse moves to a new monitor (to activate it), 60 times per second would be overkill (that makes sense if you're interacting with a interface element, like a magnifying/zoom glass).

I chose to check mouse pointer position every 100ms to ease computational burden.  Note, PointerWatcher is quite clever and only fires checks if mouse moves (and has some approaches to minimize the number of times it fires depending on movement).

Reply:
> for drag/drop across monitors

It just occurred to me that drag/drop support is no longer an issue with this approach - i.e. it was only an issue since the `enterMonitor` element was blocking it (and we've not removed the `enterMonitor` element entirely).

So this approach it only to activate the space when mouse enters a new monitor.